.github/workflows/iroha2-dev.yml

```
@@ -71,32 +63,32 @@ jobs:
       - uses: Swatinem/rust-cache@v1
       - name: Build
         run: mold --run cargo build --release --verbose
-      - name: Archive build
-        uses: actions/upload-artifact@v2
+      - name: Archive iroha
+        uses: actions/upload-artifact@v3
         with:
           name: cargo-build-release
           path: target/release/iroha
-      - name: Archive Client CLI build
-        uses: actions/upload-artifact@v2
+      - name: Archive iroha_client_cli
+        uses: actions/upload-artifact@v3
         with:
           name: cargo-client-cli-build-release
           path: target/release/iroha_client_cli
-      - name: Archive Crypto CLI build
-        uses: actions/upload-artifact@v2
+      - name: Archive kagami
+        uses: actions/upload-artifact@v3
         with:
           name: cargo-crypto-cli-build-release
           path: target/release/kagami
-      - name: Run schema generation
+      - name: Generate schema
         run: |
           mkdir -p target/schema
           cargo run --bin kagami -- schema >target/schema/schema.json
-      - name: Upload schema
-        uses: actions/upload-artifact@v2
+      - name: Archive schema
+        uses: actions/upload-artifact@v3
         with:
           name: schema
           path: target/schema
 
-  print-telemetry:
+  print_and_archive_telemetry:
```

```suggestion
  telemetry:
```
The fact that telemetry is being printed doesn't help much. The fact that it's archived, even less so.